dbt_subprojects/hourly_spellbook/models/_sector/staking/ethereum/staking_ethereum_flows.sql

```diff
@@ -113,3 +99,102 @@ SELECT w.block_time
 , NULL AS evt_index
 FROM {{source('ethereum', 'withdrawals')}} w
 INNER JOIN indexes i USING (validator_index)
+
+
+
+
+
+
+
+
+
+
+
+
+{{ config(
+    
+    schema = 'staking_ethereum',
+    alias = 'flows',
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "sector",
+                                "staking",
+                                \'["hildobby", "0xBoxer"]\') }}')
+}}
```

Comment:
@hildobby you duplicated the full model below, which uses incorrect syntax for the source (hence the compile error)